storm-multilang/dotnet/src/main/dotnet/Dotnet.Storm.Adapter/Test/TestAPI.cs

```diff
@@ -0,0 +1,78 @@
+﻿/**
```

Comment:
Header:
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT license.